edit readCourse.data

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+#-*-coding:utf-8-*-
+
+
+import os, json
+import os,inspect
+from pymongo import MongoClient
+import pandas as pd
+import logging
+
+class Course:
+	
+    def __init__(self, *args, **kwargs):
+    	for attr in args:
+		setattr(slef,attr,[])
+  	kwargs = {'UID': '', 'crn': '', 'course_num':'', 'section_num':'', 'campus':'',\
+               	  'num_credit':'', 'course_title':'', 'days':'', 'startTime':u.split('-')[1], 'endTime':u.split('-')[2],\
+               	  'cap':'', 'wl_cap':'', 'instructor_name':'', 'startDate':j.split('-')[1], 'endDate':j.split('-')[2], 'location':'', \
+               	  'attribute':''}
+  	for k,v in kwargs:
+        	setattr(slef,k,v)
+			
+
+class Instructor:
+	def __init__(self, firstName, middleName, lastName):
+		self.firstName = firstName
+		self.middleName = middleName
+		self.lastName = lastName
+		self.website = ''
+		self.email = ''
+		self.department = []
+		self.relatedCourse = []
+
+class Department:
+	def __init__(self, deptName):
+		self.deptName = deptName
+		self.courses = []
+
+class Lab:
+	def __init__(self, UID, days, startTime, endTime, \
+				startDate, endDate, instructor, location):
+		self.UID = UID
+		self.days = days
+		self.startTime = startTime
+		self.endTime = endTime
+		self.startDate = startDate
+		self.endDate = endDate
+		self.instructor = instructor
+		self.location = location
+		
+class Logger:
+	def __init__(self, __name__):
+		self.logger = logging.getLogger(__name__)
+		self.logger.setLevel(logging.DEBUG)
+		fmt = logging.Formatter('[%(asctime)s] [%(levelname)s] %(message)s', '%Y-%m-%d %H:%M:%S')
+		sh = logging.StreamHandler()
+		sh.setFormatter(fmt)
+		sh.setLevel(DEBUG)
+		fh = logging.FileHandler(path)
+		fh.setFormatter(fmt)
+		fh.setLevel(INFO)
+		self.logger.addHandler(sh)
+		self.logger.addHandler(fh)
+###not sure
+		def debug(self,message):
+		self.logger.debug(message)
+		def info(self,message):
+		self.logger.info(message)
+		def war(self,message):
+		self.logger.warn(message)
+		def error(self,message):
+		self.logger.error(message)
+		def cri(self,message):
+		self.logger.critical(message)
+		
+		
+    
+def main():
+
+	logobj = Logger(__name__)
```

Comment:
`log` or `logger` is better name. 
We don't have to create a `Logger` class
Try to use `logging.getLogger` function.